tests/assignment_comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -61,12 +123,114 @@ let f3 = (
   b //comment
 ) => {};
 
+let f4 = // Comment
+  () => {};
+
+let f5 =
+
+  // Comment
+
+  () => {}
+
+let f6 = /* comment */
+
+  // Comment
+
+  () => {}
+
+fnNumber =
+  // Comment
+  3;
+
+fnNumber =
+
+  // Comment
+
+  3;
+
+fnNumber =
+  // Comment
+  // Comment
```

Comment:
In such tests with multiple comments, to make sure that the order of comments stays the same, it's better to use comments with different content.